assemblers/org.osbuild.tar

```diff
@@ -7,13 +7,19 @@ import sys
 
 def main(tree, output_dir, options):
     filename = options["filename"]
-    compression = options["compression"]
+    compression = options.get("compression", None)
```

Comment:
the `,None` part is redundant because it is the default. Maybe we could remove it?